### PR TITLE
Don’t match `MZ` for `Cab`.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----

Fixes https://github.com/Homebrew/brew/pull/4541#issuecomment-408058867.

### CODE REVIEW

File: Library/Homebrew/test/.codecov.yml [hunk 1]
```diff
@@ -2,7 +2,9 @@ comment: off
...
   - "::Library/Homebrew/"
 coverage:
+  round: nearest
+  precision: 2
   status:
     project:
-    default:
-        threshold: 0.1
+      default:
+        threshold: 0.05%
```
Comment (MikeMcQuaid) [inline Library/Homebrew/test/.codecov.yml]: How do these changes relate to this PR?

Comment (reitermarkus) [reply Library/Homebrew/test/.codecov.yml]: The now required coverage check was failing.

Comment (MikeMcQuaid) [reply Library/Homebrew/test/.codecov.yml]: 👍

**Comment by MikeMcQuaid** [comment]
Thanks @reitermarkus!

**Pull request merged**